### DISCUSSION TITLE
Talk

### DISCUSSION THREAD

**Discussion by Altify-Development** [Altify-Developing/Altify-Developing-Main#3] [General]
Write a comment here saying what you want to see from this organization or if you would like to join and make a contribution.

**Comment by Altify-Development** [comment]
@williameom5678 yo what you good at coding wise

**Reply by williameom5678** [reply]
Javascript, C, assembly(learning)

**Reply by Altify-Development** [reply]
if you could set up an auto semantic version committing system, I would be greatly appreciative.

**Reply by Altify-Development** [reply]
plus if you have any collaborators that you think would be interested in joining please list them.

**Reply by williameom5678** [reply]
Hmm...

**Reply by Altify-Development** [reply]
i will be working on tools

**Comment by Altify-Development** [comment]
yo @williameom5678 what are your hours of activity?

**Reply by williameom5678** [reply]
only on weekends. I'm a student

**Reply by Altify-Development** [reply]
okay. im on and off between 9:00am to 9pm in PST ( pacific standard timezone )

**Comment by Altify-Development** [comment]
anyone reply to this if you want to join the organization to make hacking tools.

**Reply by wg-d** [reply]
Hi, I want to join.

**Reply by Altify-Development** [reply]
@wg-d are you still wanting to join me? if so contact me whenever you feel like on my discord server - https://discadia.com/altify

**Comment by Altify-Development** [comment]
wus good?
